src/btree/bt_sync.c

```diff
@@ -137,6 +137,57 @@ err:	WT_TRET(__wt_page_release(session, next, flags));
 }
 
 /*
+ * __sync_checkpoint_progress --
+ * 	Output a checkpoint progress message.
+ */
+static void
+ __sync_checkpoint_progress(WT_SESSION_IMPL *session)
+{
+	struct timespec cur_time;
+	WT_CONNECTION_IMPL *conn;
+	uint64_t time_diff;
+
+	conn = S2C(session);
+	time_diff = 0;
+	__wt_epoch(session, &cur_time);
+
+	/* Time since the full database checkpoint started */
+	time_diff = WT_TIMEDIFF_SEC(cur_time, conn->ckpt_verb_start_time);
```

Comment:
I can't tell without scrolling to declarations if ckpt_verb_start_time started before or after scrub phase.